extensions/sql-database-projects/src/models/project.ts

```diff
@@ -589,3 +589,5 @@ export enum SystemDatabase {
 	master,
 	msdb
 }
+
+export const reservedProjectFolders = ['Properties', 'Data Sources', 'Database References'];
```

Comment:
Actually, Properties should either be a physical folder that gets created on disk (how SSDT works), or not created at all (even in virtual form, since I've never seen the Properties folder get used for anything in SSDT, and there's no harm in deleting it from the project in SSDT).

It's be awesome if you added either of those changes to this PR :)

Reply:
Oh weird I haven't seen Properties be a physical folder that gets created by SSDT with the other folders in the project. Where does the folder get added? 

It's probably ok to not show the folder in ADS since editing the properties in the UI isn't supported, but I'm hesitant to get rid of it altogether because then it removes this launch point to editing the properties in SSDT: 
![image](https://user-images.githubusercontent.com/31145923/86041320-578c5a80-b9fa-11ea-8df2-058edbdb84b2.png)

Reply:
Yep, we can also in long run have project dashboard open from properties may be... so lets not remove completely anyways. We can stop showing in ADS tree - but I dont now its required since its not hurting any scenario.

Question on reserved folder - can one add afile in reserved folder?

Reply:
@kisantia Looks like I got my wires crossed between csproj and sqlproj again; .net framework projects create a physical properties folder.  Virtual folder it is!

Reply:
@udeeshagautam no files can't be added to these reserved folders. The database references folder isn't a physical folder (the refs are from the sqlproj) and the data sources are read from a file single datasources.json file in the project folder.